src/storm-cli-utilities/model-handling.h

```diff
@@ -441,7 +441,8 @@ std::shared_ptr<storm::models::ModelBase> buildModelDd(SymbolicInput const& inpu
 }
 
 template<typename ValueType>
-std::shared_ptr<storm::models::ModelBase> buildModelSparse(SymbolicInput const& input, storm::settings::modules::BuildSettings const& buildSettings) {
+std::shared_ptr<storm::models::ModelBase> buildModelSparse(SymbolicInput const& input, storm::settings::modules::IOSettings const& ioSettings,
```

Comment:
I understand this, but can we refactor this such that we do not pass more and more settings around. Can we do a createOptions first?

Reply:
I made a `createOptionsSparse` which makes it a bit more clear what's happening but does not really avoid passing around the settings. Any suggestions?

Reply:
I think that is a significant improvement because it isolates the settings from the rest of the logic. I would also not mind if that createOptionsSparseFromSettings would just grab the settings from the manager, rather than them being passed around? (It is a function that explicitly depends on settings, so ...)

Reply:
Yes, 100% agree. I've updated this now.